### PR TITLE
Swap `RichText`

### PR DESCRIPTION
**Pull request by estrattonbailey**
Changes between old and new `RichText`:
- text size now defaults to 14, was 15
- line height now defaults to 1.3, was 1.2

Posts in feeds and threads have no (or very very minimal) visual changes. After/before (reverse order sorry) shots:

Group 1
Group 2
Group 3
Group 4
Group 5

### CODE REVIEW

File: src/alf/atoms.ts [hunk 1]
```diff
@@ -176,43 +176,59 @@ export const atoms = {
   },
   text_2xs: {
     fontSize: tokens.fontSize._2xs,
+    letterSpacing: 0.25,
```
Comment (estrattonbailey) [inline src/alf/atoms.ts:179]: All our existing type styles have built in letter spacing, so for consistency I've added here too

File: src/alf/atoms.ts [hunk 2]
```diff
@@ -176,43 +176,59 @@ export const atoms = {
...
   },
   leading_normal: {
     lineHeight: 1.5,
   },
+  tracking_normal: {
+    letterSpacing: 0,
+  },
+  tracking_wide: {
+    letterSpacing: 0.25,
+  },
```
Comment (estrattonbailey) [inline src/alf/atoms.ts:231]: Couple more Tailwind atoms

File: src/alf/atoms.ts [hunk 3]
```diff
@@ -176,43 +176,59 @@ export const atoms = {
...
   text_5xl: {
     fontSize: tokens.fontSize._5xl,
+    letterSpacing: 0.25,
   },
   leading_tight: {
     lineHeight: 1.15,
   },
   leading_snug: {
-    lineHeight: 1.25,
+    lineHeight: 1.3,
```
Comment (estrattonbailey) [inline src/alf/atoms.ts:221]: Post text defaults to 1.3, so for consistency through, I bumped this up to match. 0.5 difference is negligible, and we hardly use this atom so far elsewhere.

File: src/components/Link.tsx [hunk 4]
```diff
@@ -82,7 +83,9 @@ export function useLink({
 
   const onPress = React.useCallback(
     (e: GestureResponderEvent) => {
-      outerOnPress?.(e)
+      const exitEarlyIfFalse = outerOnPress?.(e)
+
+      if (exitEarlyIfFalse === false) return
```
Comment (estrattonbailey) [inline src/components/Link.tsx:88]: Something I've been meaning to do, was gonna use for tags but going to do something else now. Might as well include.

File: src/components/RichText.tsx [hunk 5]
```diff
@@ -29,7 +29,7 @@ export function RichText({
   const [richText, setRichText] = React.useState<RichTextAPI>(() =>
     value instanceof RichTextAPI ? value : new RichTextAPI({text: value}),
   )
-  const styles = [a.leading_normal, style]
+  const styles = [a.leading_snug, flatten(style)]
```
Comment (estrattonbailey) [inline src/components/RichText.tsx:32]: Default's to what we use for post text. `leading_normal` (1.5) is too wide for anything but longform, and `tight` (1.15) is better for titles. `snug` (1.3) is a good default.

Previous `RichText` component defaulted to 1.2, which is a little tight imo.

**Review by haileyok — APPROVED** [review]
LGTM!

**Pull request merged**